homeassistant/components/victron_gx_mqtt/__init__.py

```diff
@@ -0,0 +1,71 @@
+"""The victron_mqtt integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from homeassistant.const import Platform
+
+if TYPE_CHECKING:
+    from homeassistant.config_entries import ConfigEntry
+    from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .hub import Hub
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [
+    Platform.SENSOR,
+]
+
+__all__ = ["DOMAIN"]
```

Comment:
We never export the domain, except to tests, but we don't have strict guidelines for that. Other than that, we only use `__all__` for things we export to other integrations